vtStor/Drive.cpp

```diff
@@ -29,7 +29,7 @@ cDrive::cDrive(std::shared_ptr<vtStor::cDeviceInterface> Device, DeviceHandle De
 
 cDrive::~cDrive()
 {
-    CloseDeviceHandle(m_DeviceHandle);
+    //CloseDeviceHandle(m_DeviceHandle); // comment by Minh Mai
```

Comment:
Could you need to comment this code? It will make vtStor works incorrectly on Win.